patches/ppc64le-and-riscv64-support.patch

```diff
@@ -211,26 +253,45 @@ index 3d6c2eba6e9..96e7707542e 100644
 +		'xdg-utils (>= 1.0.2)'
 +	],
  };
-diff --git a/build/linux/debian/install-sysroot.ts b/build/linux/debian/install-sysroot.ts
-index aa417dcc722..2e0eaa34f4f 100644
---- a/build/linux/debian/install-sysroot.ts
-+++ b/build/linux/debian/install-sysroot.ts
-@@ -151,6 +151,15 @@ export async function getVSCodeSysroot(arch: DebianArchString): Promise<string>
- 			expectedName = `arm-rpi-linux-gnueabihf${prefix}.tar.gz`;
- 			triple = 'arm-rpi-linux-gnueabihf';
- 			break;
+diff --git a/build/linux/debian/install-sysroot.js b/build/linux/debian/install-sysroot.js
+index feca7d3..59c1c6a 100644
+--- a/build/linux/debian/install-sysroot.js
++++ b/build/linux/debian/install-sysroot.js
+@@ -70,7 +70,9 @@ async function fetchUrl(options, retries = 10, retryDelay = 1000) {
+         const timeout = setTimeout(() => controller.abort(), 30 * 1000);
+         const version = '20240129-253798';
+         try {
+-            const response = await fetch(`https://api.github.com/repos/Microsoft/vscode-linux-build-agent/releases/tags/v${version}`, {
++            // If the assetName starts with 'powerpc64le', we fetch from lex-ibm/vscode-linux-build-agent
```

Comment:
Can you make a PR in VSCodium/vscode-linux-build-agent? Thx

Reply:
Oops, I haven't seen you PR... I need to buy better glasses...

Reply:
Ok, it's running https://github.com/VSCodium/vscode-linux-build-agent/actions/runs/8839861547